buffer/src/main/java/io/netty/buffer/CompositeByteBuf.java

```diff
@@ -371,6 +371,7 @@ public CompositeByteBuf removeComponents(int cIndex, int numComponents) {
         return this;
     }
 
+    @Override
     public Iterator<ByteBuf> iterator() {
         ensureAccessible();
         List<ByteBuf> list = new ArrayList<ByteBuf>(components.size());
```

Comment:
@trustin @Scottmitch https://github.com/netty/netty/issues/3623 to keep track of the improvement (will do the fix next week)